docs/examples/directives.admonitions.yml

```diff
@@ -90,22 +90,6 @@ cases:
                     - type: text
                       value: |-
                         and an example of a note admonition.
-    myst: |-
-      ```{note} This is a title in myst
-        and an example of a note admonition.
-      ```
-    html: |-
-      <aside class="admonition note">
-        <p class="admonition-title">This is a title in myst</p>
-        <p>
-          and an example of a note admonition.
-        </p>
```

Comment:
Sounds good, makes sense to defer this until there are also changes in the python ecosystem. Looking forward to that :).